index.html

```diff
@@ -207,43 +207,45 @@ <h3>Multi-post stories</h3>
         </div>
       </article>
       </section>
+    </main>
 
       <!-- about section -->
       <section class="about-container">
       <article class="about">
-        <h2 class="about-h1-b">About Myself</h2>
-
-        <h3 class="about-p-b">
-          I'm a Software developer! Can help you build a product, feature or Website Look through Some of my Work and
-          experience! if you like What you see and have a project you need coded, don't hesitate to contact me.
-        </h3>
-
-        <h3 class="connect-b">LET'S CONNECT</h3>
-
-        <ul class=" social">
-          <li>
-            <a href="https://twitter.com/ejama_isaac">
-              <img src="img/Vector (3).svg" alt="">
-            </a>
-          </li>
-          <li>
-            <a href="https://www.linkedin.com/in/ejama-isaac-bab607267/"><img src="img/Linkedin icon.svg" alt=""></a>
-          </li>
-          <li>
-            <a href="#"><img src="img/Vector (2).svg" alt=""></a>
-          </li>
-          <li>
-            <a href=""><img src="img/Vector.svg" alt=""></a>
-          </li>
-          <li>
-            <a href="https://github.com/isaac-ej"><img src="img/Vector (1).svg" alt=""></a>
-          </li>
-        </ul>
         <div>
-          <a href="#" class="btn-b active">Get my resume</a>
-        </div>
+          <h2 class="about-h1-b">About Myself</h2>
 
+          <h3 class="about-p-b">
+            I'm a Software developer! Can help you build a product, feature or Website Look through Some of my Work and
+            experience! if you like What you see and have a project you need coded, don't hesitate to contact me.
+          </h3>
+          
+          <h3 class="connect-b">LET'S CONNECT</h3>
```

Comment:
- [ ] Great job on your desktop version! I just wanted to kindly suggest aligning the headings properly. Currently, both of them appear on a single line, but following the Figma template, it would be ideal to have them spread across two lines. 😉

   ### Your design:
   ![image](https://github.com/Isaac-ej/Portfolio_Website/assets/107952724/d05da0b8-f795-49f0-8930-6527ed0bad6d)

   ### Figma design:
   ![image](https://github.com/Isaac-ej/Portfolio_Website/assets/107952724/4cadac50-da14-443e-8357-e0d580d413a1)